argus/action_manager/windows.py

```diff
@@ -218,6 +218,54 @@ def specific_prepare(self):
         # We don't have anythong specific for the base
         LOG.debug("Prepare something specific for OS Type %s", self._os_type)
 
+    def _exists(self, path, path_type):
+        """Check if the path exists and it has the specified type.
+
+        :param path:
+            Path to check it exists.
+        :param path_type:
+            This can be 'Leaf' or 'Container'
+        """
+        cmd = 'Test-Path -PathType {} -Path "{}"'.format(path_type, path)
+        stdout, stderr, exit_code = self._client.run_remote_cmd(
+            cmd=cmd, command_type=util.POWERSHELL)
+
+        return stdout.strip() == "True"
+
+    def exists(self, path):
+        """Check if the path exists.
+
+        :param path:
+            Path to check if it exists.
+        """
+        return self._exists(path, self.PATH_ANY)
+
+    def is_file(self, path):
+        """Check if the file exits.
+
+        :param path:
+            Path to check if it exists and it's a file.
+        """
+        return self._exists(path, self.PATH_LEAF)
+
+    def is_dir(self, path):
+        """Check of the directory exists.
+
+        :param path:
+            Path to check if it exists and it's a directory.
+        """
+        return self._exists(path, self.PATH_CONTAINER)
+
+    def remove_file(self, path):
+        """Remove a file."""
```

Comment:
You should check if the received path exists and it is a file.
Also you should take care of all the exceptions that may occur.